it.mijorus.smile.json

```diff
@@ -1,7 +1,7 @@
 { 
     "app-id": "it.mijorus.smile",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "41",
+    "runtime-version": "42",
```

Comment:
43 is out

Reply:
unable to push the update anyway, until I don't fix

https://github.com/flathub/flatpak-builder-lint/issues/62#issuecomment-1286588346